### ISSUE TITLE
mtconv not improve acc   compare with depthwise ，is the training data too small matter？

### ISSUE THREAD

**Issue by l2009312042** [Interlagos/TENet-kws#1]
it is a great job ,thanks for sharing ! the following is my test result , from this ,the acc is not improve ,can you give me some advice how to improve the acc ?

| data|  model| acc | size|  
|:-------:|:-------:|:----:|:----:|
|  google |net12|96.6|1.3m|
|  google|net6|96.7|785kb|
| google|net12_mt_noconvert|96.56|2.41m|
| google|net12_mt_convert|96.56|1.78m|
| google|net6_mt_noconvert|96.75|1.31m|
| google|net6_mt_convert|96.75|0.99m|

**Comment by Interlagos** [comment]
Thanks for your attention to our work. 

We recommend training TENet with the default hyper-parameters in this repo, especially the weight decay is set to 4e-5 instead of 1e-4, which affects the result a lot.

We retrain the following models 10 times, respectively. Here is the result:

|       | TENet12_MTConv |   TENet12   | TENet6_MTConv |   TENet6    |
| :---: | :------------: | :---------: | :-----------: | :---------: |
| no.1  |    96.7543     |   96.6569   |    96.7218    |   96.5920   |
| no.2  |    96.9815     |   96.4946   |    96.7868    |   96.3648   |
| no.3  |    97.1438     |   96.3973   |    96.6245    |   96.3973   |
| no.4  |    96.8517     |   96.2025   |    96.2999    |   96.4946   |
| no.5  |    96.8814     |   96.5271   |    96.2350    |   96.4622   |
| no.6  |    96.5920     |   96.8192   |    96.6569    |   96.7868   |
| no.7  |    96.5324     |   96.8841   |    97.1762    |   96.2674   |
| no.8  |    96.6569     |   96.4622   |    96.4297    |   96.6245   |
| no.9  |    97.0464     |   96.5271   |    96.5271    |   96.5271   |
| no.10 |    96.6569     |   96.4946   |    96.7543    |   96.5271   |
| mean  |  **96.8097**   | **96.5466** |  **96.6212**  | **96.5044** |

Hope it can help you.

**Comment by l2009312042** [comment]
thanks for your  advice
